cli/ceph/orch/tuned_profile.py

```diff
@@ -0,0 +1,70 @@
+from cli import Cli
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class TunedProfile(Cli):
+    def __init__(self, nodes, base_cmd):
+        super(TunedProfile, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} tuned-profile"
+
+    def apply(self, spec_file, **kw):
+        """
+        apply tuned profile.
+        Args:
+            instance: cluster instance
+            config: key/value pairs that needs to be provided to the installer
+            host_obj : host objects
+        """
+        cmd = f"{self.base_cmd} apply -i {spec_file}"
+        out = self.execute(sudo=True, cmd=cmd)
+        log.info(out)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def list(self):
+        """
+        list tuned profile.
+        Args:
+            instance: cluster instance
+            specs: tuned profile specs
+            cfg : key/value pairs that needs to be provided to the installer
+        """
+        cmd = f"{self.base_cmd} ls"
+        out = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].split("\n")
+        return out[0]
+
+    def remove(self, **kw):
+        """
+        remove tuned profile.
+        Args:
```

Comment:
Same as above

Reply:
Updated

Reply:
Same as above - Remove this key if no optional parameters are supported.